examples/ollama/devenv.nix

```diff
@@ -0,0 +1,13 @@
+{ ... }:
```

Comment:
We have hit the limit of examples as github jobs, so we have to remove it for now.

Reply:
removed, thank you